login-fire-common-behavior.html

```diff
@@ -154,6 +154,16 @@
       },
 
       /**
+        * Whether to show the view at sign up or sign in
+        *
+        * @type {Boolean}
+        */
+        showSignUp: {
```

Comment:
Not planned yet. The idea wasn't to distinguish the action but just the text. There are still some who do not understand that "Sign in with..." would sign you up if you don't have an account, so if you wanted to add that distinction in the text you can. You can also override the text and make it always "Sign in with..." if you want.

I think the user should be able to sign in with a provider if they haven't signed up before. We can move this to an issue and have further discussion on it if you want.